tables/regression_table_full_dom.tex

```diff
@@ -2,7 +2,7 @@
 \toprule
 {} &                                       (1) &                                       (2) \\
 \midrule
-Dependent Var                             &               $${\it VShare}^{\it ulti}$$ &               $${\it VShare}^{\it betw}$$ \\
+Dependent Var                             &               ${\it VShare}^{\it ulti}$ &               ${\it VShare}^{\it betw}$ \\
```

Comment:
the .tex is automatically generated from code
please correct the code instead of the output @lyc0603 thanks

Reply:
Created a new function to avoid duplicated $$